docs/docs/config/system.md

```diff
@@ -69,8 +69,13 @@ Exactly zero or one of the following options may be set to `y`. The first is use
 | `CONFIG_USB_DEVICE_MANUFACTURER`  | string | The manufacturer name advertised to USB | `"ZMK Project"` |
 | `CONFIG_USB_HID_POLL_INTERVAL_MS` | int    | USB polling interval in milliseconds    | 1               |
 | `CONFIG_ZMK_USB`                  | bool   | Enable ZMK as a USB keyboard            |                 |
+| `CONFIG_ZMK_USB_BOOT`             | bool   | Enable USB Boot protocol support        | n               |
 | `CONFIG_ZMK_USB_INIT_PRIORITY`    | int    | USB init priority                       | 50              |
 
+:::note USB Boot protocol support
+
+By default USB Boot protocol support is disabled, however in certain situations such as the input of Bitlocker pins or FileVault passwords may require it to be enabled.
+
```

Comment:
FYI this is broken, see https://deploy-preview-2107--zmk.netlify.app/docs/config/system#usb. Below might fix it, but you can make sure by [testing it locally](https://zmk.dev/docs/development/documentation#testing-documentation-updates-locally) or checking the deploy preview in the PR checks after pushing changes.

```suggestion
:::note USB Boot protocol support

By default USB Boot protocol support is disabled, however certain situations such as the input of Bitlocker pins or FileVault passwords may require it to be enabled.

:::
```

Reply:
Thank you for pointing that feature out to me! The preview now looks good to me, unfortunately I don't know why the pre-commit check fails.

Reply:
No worries, it was missing a newline after the end. Added it and merged.